mercury_app/tests.py

```diff
@@ -717,10 +741,13 @@ def test_get_summary_types_handed_one(self):
                        'name': 'Gorra handed', 'id': 1},
                       {'quantity': 0.0, 'percentage': 0.0,
                        'name': 'Gorra don\'t handed', 'id': 2}]])
-        MerchandiseFactory(delivered=True,
-                           name='Gorra',
-                           quantity=1,
-                           order=self.order)
+        mercha = MerchandiseFactory(name='Gorra',
+                                    quantity=1,
+                                    order=self.order)
+        TransactionFactory(
+            merchandise=mercha,
+            operation_type='HA',
```

Comment:
It would be great if you can use constants instead of repeating same string over again